<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Digitizer_Configuration</class>
 <widget class="QDialog" name="Digitizer_Configuration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>660</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>660</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>660</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Acquisition Configuration</string>
  </property>
  <widget class="QPushButton" name="pushButton_DigConfig_OK">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>610</y>
     <width>92</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_DigConfig_Cancel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>610</y>
     <width>92</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_DigConfig_Load">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>610</y>
     <width>131</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Load settings</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_DigConfig_Save">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>610</y>
     <width>131</width>
     <height>36</height>
    </rect>
   </property>
   <property name="text">
    <string>Save settings</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>General settings</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>300</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Settings for selected channel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>470</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Channel Enable Mask</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_ChannelEnableMask">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>470</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>DISABLE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ENABLE</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Record Length</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_RecordLength">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>430</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Post Trigger Size (%)</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_PostTriggSize">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>80</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Acquisition Mode</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_AcquisitionMode">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>120</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>SOFTWARE CONTROLLED</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>S-IN CONTROLLED</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>FIRST TRG CONTROLLED</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_Channels">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>360</y>
     <width>121</width>
     <height>36</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Channel 0</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Channel 1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Channel 2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Channel 3</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Channel 4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Channel 5</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Channel 6</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Channel 7</string>
    </property>
   </item>
  </widget>
  <widget class="QCheckBox" name="checkBox_EveryChannel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>241</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Same settings for every channel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>510</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Channel DC Offset</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox_ChannelDCOffset">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>510</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>160</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Dual Edge Sampling Mode</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>199</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Decimation Factor</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_DESM">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>160</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ENABLE</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>DISABLE</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="comboBox_DecimationFactor">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>200</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>1</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>32</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>64</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>128</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
